test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -339,6 +339,13 @@ object TestAspect extends TimeoutVariants {
   val jvmOnly: TestAspectPoly =
     if (TestPlatform.isJVM) identity else ignore
 
+  /**
+   * An aspect that repeats the test a default number of times, ensuring it is
+   * stable ("non-flaky"). Stops at the first failure.
+   */
+  val nonFlaky: TestAspectPoly =
+    nonFlaky(100)
```

Comment:
Not that I don't appreciate this change, far from it, but why didn't you use default arguments, of course, in the original function declaration, like so:

`def nonFlaky(n0: Int = 100): TestAspectPoly = ???`

Reply:
If you do that then you would need to call the method as `nonFlaky()` instead of `nonFlaky`. That is somewhat irregular (no other test aspect is called that way and it gives the impression that it is possibly an impure method) so I think it is nicer to do it this way, especially when it is only a few extra lines of code.

Reply:
Well, you answered my question 😄